getips.py

```diff
@@ -1,17 +1,30 @@
-#!/usr/bin/env python
-
-import urllib.request
-import json
-
-with urllib.request.urlopen("https://ip-ranges.amazonaws.com/ip-ranges.json") as url:
-    data = json.loads(url.read().decode())
-    # Print all AWS IPs
-    allawsips= list(ip_prefix["ip_prefix"] for ip_prefix in data["prefixes"])
-    with open('awsips.txt', 'w') as f:
-        for cidr in allawsips:
-            f.write("%s\n" % cidr)
-    # Print all Cloudfront IPs
-    cloudfront = list(ip_prefix["ip_prefix"] for ip_prefix in data["prefixes"] if "CLOUDFRONT" in ip_prefix["service"])
-    with open('cloudfront.txt', 'w') as f:
-        for cidr in cloudfront:
-            f.write("%s\n" % cidr)
\ No newline at end of file
+#!/usr/bin/env python
+
+# Standard Python Libraries
+import ipaddress
+import json
+import urllib.request
+
+
+def output_cidrs(filename: str, cidrs: list[str]):
+    """Output the given CIDRs to the given file."""
+    with open(filename, "w", encoding="utf-8") as out:
+        for cidr in cidrs:
```

Comment:
Go open source! You saved me from telling myself the lie of "I'll make it work and clean-up the code later"